Gemfile

```diff
@@ -5,6 +5,8 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in rubocop-solidus.gemspec
 gemspec
 
+gem 'bump'
```

Comment:
Are you sure that this change is related to this commit? 🤔

Reply:
Yes, because `bump` is used in the [`cut_release` rake task](https://github.com/nebulab/rubocop-solidus/blob/86a35199afd28091af604bd1e710e79e1d651197/tasks/cut_release.rake#L64)

Reply:
I was confused because the commit message said that this should import the rubocop rake tasks.